packages/app/src/files-manager/components/files-manager/files-manager.component.tsx

```diff
@@ -58,11 +58,16 @@ const FilesManager: FunctionComponent<FilesManagerProps> = ({
           ...element,
           size: free,
         }
-      }
-      if (element.type === DiskSpaceCategoryType.UsedSpace) {
+      } else if (element.type === DiskSpaceCategoryType.UsedSpace) {
+        return {
+          ...element,
+          size: reservedSpace,
+        }
+      } else if (element.type === DiskSpaceCategoryType.Music) {
         return {
           ...element,
-          size: systemMemory,
+          size: usedUserSpace,
```

Comment:
I think that music size should be a the sum of all music file

Reply:
Changed in [2f838ad](https://github.com/mudita/mudita-center/pull/915/commits/2f838ad5888146b962ea0c53a4419615d2c45ec8)